make/compile

```diff
@@ -28,19 +29,19 @@ _cache() {
 
     for package_version in $(get_package_version_list) ; do
         package_hash="${package_version##*/}"
-        archive="${SCF_PACKAGE_COMPILATION_CACHE}/${package_hash}.tar.xz"
+        archive="${SCF_PACKAGE_COMPILATION_CACHE}/${STEMCELL_HASH}/${package_hash}.tar.xz"
         test -e "${archive}" && {
             # Don't recreate an archive that already exists
             echo "Found:   ${archive}"
             continue
         }
-        test -d "${FISSILE_WORK_DIR}/compilation/${package_hash}/compiled" || {
+        test -d "${FISSILE_WORK_DIR}/compilation/${STEMCELL_HASH}/${package_hash}/compiled" || {
             echo "Missing: ${package_hash}/compiled"
```

Comment:
Should we add the stemcell-hash to the output as well ?
When I looked at the log I was confused at first, because I thought it was looking at and reporting some old directory.

Reply:
We could; I was thinking that might be too much text (which doesn't change anyway), pushing the thing too wide for display in things like Jenkins.

Reply:
Ok.